examples/constant_liar.py

```diff
@@ -0,0 +1,31 @@
+"""
+Example for parallel optimization with skopt.
+The points to evaluate in parallel are selected according to the "constant lie" approach.
+
+"""
+
+import numpy as np
+from multiprocessing.pool import ThreadPool
+from skopt.space import Real
+from skopt.learning import GaussianProcessRegressor
+from skopt import Optimizer
+
+# ThreadPool is used for parallel computations
+pool = ThreadPool()
```

Comment:
(thinking about the rest so just a small comment) can we ditch this as with python this won't actually run in parallel and for the example it should be ok to illustrate the idea of getting several points, evaluating them, then telling them without having to add actual parallel execution. (or we should use the process pool)